review/tests/test_review_manager.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from datetime import timedelta
+from django.utils import timezone
+from review.models import Review
+
+
+@pytest.fixture
+def reviews():
+    now = timezone.now()
+    # Checks if the reviews were created already
+    review1 = Review.objects.filter(client_id='user4', professional_id='pro1')
+    review2 = Review.objects.filter(client_id='user5', professional_id='pro2')
+    review3 = Review.objects.filter(client_id='user6', professional_id='pro3')
+    # Now there will be up to one review in each 'review' variable
+    review1 = get_single_reviews(review1, rating='5', description='Excellent!', date_posted=now - timedelta(days=3),
+                                 client_id='user4', professional_id='pro1')
+    review2 = get_single_reviews(review2, rating='4', description='Good', date_posted=now - timedelta(days=1),
+                                 client_id='user5', professional_id='pro2')
+    review3 = get_single_reviews(review3, rating='3', description='Average', date_posted=now - timedelta(days=2),
+                                 client_id='user6', professional_id='pro3')
+    yield
+    tear_down(review1, review2, review3)
+
+
+def get_single_reviews(review, rating, description, date_posted, client_id, professional_id):
```

Comment:
Is there a better way to test get single review?

Reply:
> Is there a better way to test get single review?

Fixed